Lombiq.Privacy.Samples/Recipes/Lombiq.Privacy.Samples.Content.recipe.json

```diff
@@ -0,0 +1,504 @@
+                      "Owner": "",
+                      "Author": "admin",
+                      "PrivacyConsentCheckboxPart": {
+                        "ConsentCheckbox": false,
+                        "ShowAlways": true
+                      },
+                      "FlowMetadata": {
+                        "Alignment": 3,
+                        "Size": 100
+                      }
+                    },
+                    {
+                      "ContentItemId": "4qn4t1wngqc5j4vvggk1ngae5e",
+                      "ContentItemVersionId": null,
+                      "ContentType": "Button",
+                      "DisplayText": null,
+                      "Latest": false,
+                      "Published": false,
+                      "ModifiedUtc": "2022-05-12T19:00:59.8155113Z",
+                      "PublishedUtc": null,
+                      "CreatedUtc": null,
+                      "Owner": "",
+                      "Author": "admin",
+                      "FormInputElementPart": {
+                        "Name": null
+                      },
+                      "FormElementPart": {
+                        "Id": null
+                      },
+                      "ButtonPart": {
+                        "Text": "Register",
+                        "Type": "submit"
+                      },
+                      "FlowMetadata": {
+                        "Alignment": 3,
+                        "Size": 100
+                      }
+                    }
+                  ]
+                },
+                "FlowMetadata": {
+                  "Alignment": 3,
+                  "Size": 100
+                }
+              }
+            ]
+          }
+        }
+      ]
+
+    },
+    {
+      "name": "Settings",
+      "RegistrationSettings": {
+        "UsersCanRegister": 1,
+        "UsersMustValidateEmail": false,
+        "UsersAreModerated": false,
+        "UseSiteTheme": false,
+        "NoPasswordForExternalUsers": false,
+        "NoUsernameForExternalUsers": false,
+        "NoEmailForExternalUsers": false,
+        "UseScriptToGenerateUsername": false,
+        "GenerateUsernameScript": "/* Uncomment to map AzureAD\r\n// Uncomment to output the context variable in the logs\r\n// log(\"warning\", JSON.stringify(context));\r\nswitch (context.loginProvider) {\r\n    case \"AzureAD\":\r\n        context.userName = \"azad\" + uuid();\r\n        break;\r\n    default:\r\n        log(\"Warning\", \"Provider \" + context.loginProvider + \" was not handled\");\r\n        // Uncomment to generate a username as a uuid\r\n        // context.userName = \"u\" + uuid();\r\n        break;\r\n}\r\n*/\r\n"
+      }
+    },
+    {
+      "name": "Settings",
+      "HomeRoute": {
+        "Action": "Display",
+        "Controller": "Item",
+        "Area": "OrchardCore.Contents",
+        "contentItemId": "4fxbwpz7p0gqntyxxd31zmqk4p",
+        "jsonPath": ""
+      }
+    }
```

Comment:
Is it a good idea to alter such a core setting? Now after running the recipe the site behaves more differently then you'd expect.

Reply:
I don't really understand this. It's just the home page, you can still access other pages without interacting with this form. So why can't you access this page from the menu or via direct link instead of the home page?